packages/frontend/src/components/NavBar/GlobalSearch/SearchIcon.tsx

```diff
@@ -1,37 +1,44 @@
+import { SavedChartSearchResult } from '@lightdash/common';
 import {
     Icon123,
     IconAbc,
     IconAppWindow,
-    IconChartAreaLine,
     IconFolder,
     IconLayoutDashboard,
     IconTable,
 } from '@tabler/icons-react';
 import { FC } from 'react';
-import MantineIcon, { MantineIconProps } from '../../common/MantineIcon';
+import { getChartIcon, IconBox } from '../../common/ResourceView/ResourceIcon';
```

Comment:
I think it's time we move out ResourceIcon out from ResourceView to common/ResourceIcon and extend it with additional types field, table, page. otherwise it's getting tangled

Reply:
sounds like a plan!

Reply:
@joaoviana  ideally ResourceIcon should contain all of that, since it seems like we are starting to use ResourceIcon in other places too - it will be good for consistency.
^ this can be done in the separate PR. no need to implement it here 👌

Reply:
agree 100% @IrakliJani , I will open a PR as soon as this gets done 👍